src/shims/ApiCompatBaseline.netfx.netstandard20.txt

```diff
@@ -0,0 +1,215 @@
+Compat issues with assembly mscorlib:
```

Comment:
It's created simply by building with netfx target group. Not changed by me. I guess at some point we can modify apicompat.proj to not create it.